apps/twap/src/app/conditionalOrder/getConditionalOrderId.ts

```diff
@@ -47,37 +31,35 @@ export const TWAP_HANDLER_ADDRESS: Record<SupportedChainId, string> = {
   5: twapHandlerAddress,
 };
 
-export function twapOrderToStruct(order: TWAPOrder): TWAPOrderStruct {
-  return {
-    sellToken: order.sellAmount.currency.address,
-    buyToken: order.buyAmount.currency.address,
-    receiver: order.receiver,
-    partSellAmount: order.sellAmount
-      .divide(order.numOfParts)
-      .quotient.toString(),
-    minPartLimit: order.buyAmount.divide(order.numOfParts).quotient.toString(),
-    t0: order.startTime,
-    n: order.numOfParts,
-    t: order.timeInterval,
-    span: order.span,
-    appData: order.appData,
-  };
-}
-
+/**
+ * Returns the conditional order params struct for a TWAP order.
+ *
+ * @param chainId Chain ID of the order. Must be a SupportedChainId.
+ * @param twapOrderData OrderStruct instance of the order.
+ * @returns ConditionalOrderParams instance of the order.
+ */
 export function buildTwapOrderParamsStruct(
   chainId: SupportedChainId,
-  twapOrderData: TWAPOrderStruct
+  twapOrderData: OrderStruct
 ): ConditionalOrderParams {
   return {
     handler: TWAP_HANDLER_ADDRESS[chainId],
-    salt: '0x00000000000000000000000000000000000000000000000000000018920d8ce7', // hexZeroPad(Buffer.from(Date.now().toString(16), 'hex'), 32),
+    // This salt must match between FE and BE to produce the same id, so we need to sync it.
```

Comment:
In that case, would it be better to move it somewhere else, for example cow-sdk, and then import it?